resources/src/schoolbot.py

```diff
@@ -66,106 +74,42 @@
-    await client.process_commands(message)
-
-#6.2 on_guild_join: Send tutorial to guild owner how to set up the bot
-@client.event
-async def on_guild_join(guild):
-    owner = guild.owner
-    await owner.send(content=None, embed=discord.Embed(title="Thanks for adding schoolbot to your Server!", description="To get started, you have to set up your language. Before this, nothing will work.", colour=discord.Colour.orange()).add_field(name="Start", value="Type `§setlanguage` followed by your language in small letters (e.g. `§setlanguage german`) in a channel the Bot is allowed to read and write in.", inline=False).add_field(name="Troubleshooting", value="I am a junior developer and do not know everything, so the language choice is not reliable nor stable. Please contact me by sending a private message to the bot."))
-
-#6.3 on_ready
-@client.event
-async def on_ready():
-    print("Ready")
-
-
-
-
-
-#7 Command methods
-
-#7.
-
-
-
-
-
-#8 Commands
-
-#8.1 setlanguage
-@client.command(aliases = ['changelanguage', 'cl', 'language'])
-async def setlanguage(ctx, pLanguage):
-    global langs
-    if pLanguage == "german" or pLanguage == "english":
-        langs[str(ctx.guild.id)] = pLanguage
-        with open("../data/usr/lang.json", "w") as langfile:
-            json.dump(langs, langfile, indent=4)
-        await ctx.send("Language has been set up successfully! Your language: `"+pLanguage+"`")
-    else:
-        await ctx.send("Your language is not available. Available languages are: `german`, `english`")
-
-#8.2 setprefix
-@client.command(aliases = ['changeprefix', 'cp', 'prefix'])
-async def setprefix(ctx, pPrefix):
-    global prefix
-    custom_prefixes[str(ctx.guild.id)] = pPrefix
-    with open("../data/usr/prefix.json", "w") as prefixfile:
-        json.dump(custom_prefixes, prefixfile, indent=4)
-    await ctx.send("Your prefix is now `"+custom_prefixes[str(ctx.guild.id)]+"`")
-
-#8.x test
+#6 Load specific cogs
 @client.command()
-async def test(ctx):
-    global lang
-    await ctx.send(lang["0"])
+async def load(ctx, extension):
+    if extension+".py" in os.listdir("./core"):
+        client.load_extension(f"core.{extension}")
+    if extension+".py" in os.listdir("./events"):
+        client.load_extension(f"events.{extension}")
+
+        
+        
+# 7 unload specific cogs
+@client.command()
+async def unload(ctx, extension):
+    if extension+".py" in os.listdir("./core"):
+        client.unload_extension(f"core.{extension}")
+    if extension+".py" in os.listdir("./events"):
+        client.unload_extension(f"events.{extension}")
 
 
+        
+#8 Activating/loading all cogs on startup
+for filename in os.listdir("./core"):
+    if filename.endswith(".py"):
+        client.load_extension(f"core.{filename[:-3]}")
+for filename in os.listdir("./events"):
+    if filename.endswith(".py"):
+        client.load_extension(f"events.{filename[:-3]}")
 
 
 
 #9 Token
-
-client.run("(YOUR TOKEN HERE)")
```

Comment:
Can you please stop publishing your tokens?

Reply:
I do not think I will be able to do that... I really try but wont do...